src/core/index.ts

```diff
@@ -31,6 +31,14 @@ export enum EventType {
     PAGE_VIEWED = 'k.pageViewed'
 }
 
+export enum PromptTypeName {
+    BELL = 'bell',
+    ALERT = 'alert',
+    BANNER = 'banner'
+}
+
+export const PROMPT_TYPE_NAMES = [PromptTypeName.BELL, PromptTypeName.ALERT];
```

Comment:
Missing banner?

Reply:
(actually also is this type needed / can we use the enum instead?)

Reply:
Ahh unused, will remove!